errata-email-notifications.py

```diff
@@ -0,0 +1,247 @@
+# -*- coding: utf-8 -*-
+import argparse
+import json
+import logging
+import os
+import smtplib
+import ssl
+import sys
+
+from datetime import datetime
+from email.message import EmailMessage
+from string import Template
+from urllib import request
+
+# For simplicity, we store files in the same folder as the script lives in.
+# These files are:
+#  * [distribution]_last_processed_ts: One file per distribution
+#  * app-passwd: GMail application password
+# For this reason, we'll use BASEPATH anytime we read/write files and
+# can be changed in the future if needed
+BASEPATH = os.path.dirname(__file__)
+
+DISTRIBUTIONS_ERRATA_URL = {
+    'almalinux-8': 'https://errata.almalinux.org/8/errata.json',
+    'almalinux-9': 'https://errata.almalinux.org/9/errata.json'
+}
+
+EMAIL_FROM_NAME = 'AlmaLinux Errata Notifications'
+# We create the templates for both subject and content beforehand
+SUBJECT_TEMPLATE = Template('[$errata_type Advisory] $errata_title')
+# TODO: Read content template from file?
+CONTENT_TEMPLATE = Template(
```

Comment:
I guess better to move content template to a separate file

Reply:
Dome